org.eclipse.xtext.xbase.lib.gwt/pom.xml

```diff
@@ -0,0 +1,145 @@
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.eclipse.xtext</groupId>
+		<artifactId>xtext-lib</artifactId>
+		<version>2.30.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>org.eclipse.xtext.xbase.lib.gwt</artifactId>
+	<packaging>eclipse-plugin</packaging>
+
+	<name>Eclipse Xbase GWT Library</name>
+	<description>The runtime library for the Xbase programming language.</description>
+
+	<dependencies>
+		<dependency>
+			<groupId>com.google.code.findbugs</groupId>
+			<artifactId>jsr305</artifactId>
```

Comment:
Is this a transitive dependency? Can we make it compile-only?

Reply:
In that project, I guess it's a copy and paste mistake, because the old build.gradle file did not mention that.
I'll try to remove that and see if everything still works.

Reply:
It was present also in xtend.lib.gwt. Looks like it's not needed at all (locally the build is green). I removed it from both projects: https://github.com/eclipse/xtext-lib/pull/501/commits/43171e0bcfdab7c5f35aaa06f501ab3741695026